charts/sn-platform/templates/grafana/grafana-deployment.yaml

```diff
@@ -97,7 +97,7 @@ spec:
         - name: GRAFANA_DOMAIN
           value: {{ template "pulsar.control_center_domain" . }}
         - name: GRAFANA_ROOT_URL
-          value: {{ template "pulsar.control_center_url" . }}{{ template "pulsar.control_center_path.grafana" . }}/
+          value: https://{{ template "pulsar.control_center_domain" . }}{{ template "pulsar.control_center_url" . }}{{ template "pulsar.control_center_path.grafana" . }}/
```

Comment:
I'm not sure if this schema should use `https`, if the customer uses ingress to expose a public host, ideally it should enable https